server/src/main/java/com/cloud/network/as/AutoScaleManagerImpl.java

```diff
@@ -327,7 +327,7 @@ protected AutoScaleVmProfileVO checkValidityAndPersist(AutoScaleVmProfileVO vmPr
         }
 
         if (csUrl == null || csUrl.contains("localhost")) {
-            throw new InvalidParameterValueException("Global setting endpointe.url has to be set to the Management Server's API end point");
+            throw new InvalidParameterValueException("Global setting endpoint.url has to be set to the Management Server's API end point");
```

Comment:
@GabrielBrascher, instead of only fixing the typo, we could use `ApiServiceConfiguration.ApiServletPath.key()` wherever it's referenced.

Reply:
@GutoVeronezi I've updated `AutoScaleManagerImpl.java` and `LoadBalancingRulesManagerImpl.java` to be aligned with your suggestions.
Thanks for the review!